### PR TITLE
Basic documentation

### PR DESCRIPTION
**Pull request by miso-belica**
Should close https://github.com/miso-belica/sumy/issues/55#issuecomment-196104152 and close https://github.com/miso-belica/sumy/issues/132

@syedsadiqalinaqvi it would help me if you find some time and tell me what is not clear here. It's very basic but it should help a little as a base which can be improved incrementally :)

### CODE REVIEW

**Comment by syedsadiqalinaqvi** [comment]
Woohoo, you did it! 🎉 Let me give it a read!

File: docs/summarizators.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+This is the simplest real-world algorithm. It's the first one known and it's based on the assumption that the most important sentences are those with the most significant words. The significant words are those which are more often in the text but at the same time, they don't belong among stop-words. That's why if you want to use this one you need a list of stop-words for your language. Without it, it would probably produce really bad results.
+
+## [Edmundson](http://dl.acm.org/citation.cfm?doid=321510.321519) - heuristic method with previous statistic research
+This one is the enhancement of the Luhn method mentioned previously. Edmundson added 3 more heuristics to the method to measure the importance of the sentences. He finds so-called _pragmatic words_, the words that are in headings and the position of the extracted terms. Therefore this method has 4 sub-methods and the proper combination of them results in the Edmundson method. The important part is that this method is the most **language-dependent** because it needs the list of **bonus words** and **stigma words** except for the stop-words (called the **null words** here).
```
Comment (syedsadiqalinaqvi) [inline docs/summarizators.md]: One line definitions of bonus words and stigma words, or better yet an example, would be really useful here as Edmunson tends to provide promising results and these lists can further improve generated summaries.

Comment (miso-belica) [reply docs/summarizators.md]: I added some hints from Edmundson and few examples that came to my mind :) I hope it's OK like this.

**Comment by miso-belica** [comment]
@syedsadiqalinaqvi Thanks, I add the proposed changes. Check them and let me know :)

**Comment by syedsadiqalinaqvi** [comment]
Looks great!

**Pull request merged**